scenarios/kubernetes_e2e_test.py

```diff
@@ -28,6 +28,17 @@
 
 import kubernetes_e2e
 
+FAKE_SCM = 'STABLE_BUILD_GIT_COMMIT 599539dc0b99976fda0f326f4ce47e93ec07217c\n' \
+'STABLE_BUILD_SCM_STATUS clean\n' \
+'STABLE_BUILD_SCM_REVISION v1.7.0-alpha.0.1320+599539dc0b9997\n' \
```

Comment:
Since you have real values here, it would be great if the test actually asserted the expected value of the `SCM_REVISION` that gets extracted (to make sure that the regex doesn't accidentally pick up leading/trailing whitespace, newlines, etc).